dkc/core/tests/test_file_rest.py

```diff
@@ -54,7 +55,8 @@ def test_file_rest_cannot_update_size(admin_api_client, file):
 
 
 @pytest.mark.django_db
-def test_file_rest_set_blob(admin_api_client, pending_file, s3ff_field_value):
+def test_file_rest_set_blob(admin_api_client, pending_file, s3ff_field_value, mocker):
+    mocker.patch.object(file_compute_sha512, 'delay')
```

Comment:
```suggestion
    compute_sha512_spy = mocker.spy(file_compute_sha512, 'delay')
```

This will allow you to also assert below:
```
compute_sha512_spy.assert_called_once_with(pending_file.pk)
```

Reply:
Doesn't `spy` also call the real method? That's what I'm trying to avoid since we don't want to invoke celery tasks in testing.